theme_kaizen/templates/snippets/breadcrumbs.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template id="snippet_breadcrumbs" name="Breadcrumbs">
+        <!-- Breadcrumbs -->
+        <section class="breadcrumbs breadcrumbs--bordered">
+            <div class="container">
+                <a href="#">Home</a>
+                <img
+                    class="angle-icon r-270"
+                    src="/theme_kaizen/static/img/icons/angle-icon.svg"
+                    alt="angle-icon"
+                />
+                <a href="#">Community</a>
```

Comment:
Having to edit this by hand for every page could be a bit tiring, although the layout looks good.

Perhaps a ROADMAP item to integrate with [website_breadcrumbs](https://github.com/OCA/website/pull/686/files#diff-48095d716e4f9e8e0db4ba5a2e963d58691d3d54b508d388bc67fb17136809fdR72) once it's merged?